nncf/experimental/onnx/graph/onnx_graph.py

```diff
@@ -0,0 +1,77 @@
+import onnx
+
+
+# pylint: disable=no-member
```

Comment:
isn't it too much for the whole file?
the same for nncf_graph_builder.py
I'd add this for the particular place in the code.

Reply:
The problem is that pylint abuses almost any getting attribute of onnx module.
Because there are many places where I  address such attributes I prefer to disable the pylint rule at the begging of the file.